documentation/API_documentation/device-reachability-status-API-Readiness-Checklist.md

```diff
@@ -1,12 +1,12 @@
 # API Readiness Checklist
 
-Checklist for device-reachability-status 1.0.0 in r2.2.
+Checklist for device-reachability-status 1.1.0-rc.2 in r1.1.
```

Comment:
For #9 update comment to "na for rc version"
For #13 update row to link to https://lf-camaraproject.atlassian.net/wiki/x/2wHWB